test/quantization/core/test_quantized_op.py

```diff
@@ -1262,7 +1262,15 @@ def test_max_pool2d(self, X, kernel, stride, dilation, padding, ceil_mode):
             "nn.functional": torch.nn.functional.max_pool2d,
             "nn.quantized.functional": torch.nn.quantized.functional.max_pool2d
         }
+        ops_under_test_with_indices = {
+            "nn.functional with indices": torch.nn.functional.max_pool2d_with_indices,
+        }
 
+        for name, op in ops_under_test.items():
```

Comment:
looks like we are using `ops_under_test` in both loops, do you mean to use `ops_under_test_with_indices` for one of them?